### ISSUE TITLE
Adding BirdWeather API Token crashed my docker container

### ISSUE THREAD

**Issue by have-no-clue-what-im-doing** [tphakala/birdnet-go#229]
About 20 minutes after adding my BirdWeather API Token to the config (Through the web GUI) my container crashed. When I went to restart the container I got this error:
`Error loading settings: error initializing viper: fatal error reading config file: While parsing config: yaml: cannot decode !!str my_birdweather_api_token as a !!int`

Did BirdWeather tokens use to be only int based?

I edited the config file and changed `!!int` to `!!str`
Started the container back up and all is running well.

I think the container crashed only when it detected a bird and was trying to upload it to BirdWeather and trying to use the API key. It's working perfectly fine now and is properly uploading data to BirdWeather

**Comment by tphakala** [comment]
How did you enter token the first time, through web ui or did you edit config.yaml with editor like nano or vim?

**Comment by florisre** [comment]
I am experiencing the same issue. I first entered the api key using the web interface. This inserted it into the ID field in the config file, but with the `!!int` prefix, as mwntioned before.

**Comment by have-no-clue-what-im-doing** [comment]
> How did you enter token the first time, through web ui or did you edit config.yaml with editor like nano or vim?

Yeah I updated through the web GUI. I am using the latest non dev docker image.

**Comment by tphakala** [comment]
Okay, it is a bug in web ui settings then, I'll fix this for next release.

**Comment by emilic14** [comment]
Same issue here ^^

**Comment by tphakala** [comment]
Please try with latest dev image

**Comment by emilic14** [comment]
I actually just built it and had the issue when using the web ui.
I've rebuilt the image but added the token in the config yaml, and that seems to be working ok (at least no crash on the container)

**Comment by hoover67** [comment]
I'm having the same issue with the release from June 9th. Where can I find the latest "dev image"? Thanks! 

Uwe

**Comment by florisre** [comment]
> I'm having the same issue with the release from June 9th. Where can I find the latest "dev image"? Thanks!
> 
> Uwe

Try here: https://github.com/tphakala/birdnet-go/pkgs/container/birdnet-go/266615682?tag=dev

**Comment by tphakala** [comment]
I believe this issue has been resolved
